phpunit.xml.dist

```diff
@@ -52,6 +52,7 @@
         <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener">
             <arguments>
                 <array>
+                    <element key="time-sensitive"><string>Symfony\Component\Console</string></element>
```

Comment:
Needs to be changed like in https://github.com/symfony/symfony/blob/master/phpunit.xml.dist#L59